Automattic/wp-calypso
# VideoPress: remove choose a plan step

Fixes 1utomattic/greenhouse#1667

## Proposed Changes

This PR removes the choose a plan step and enforces the monthly premium plan in the cart (as it could easily be upgraded to a yearly plan on checkout).

## Testing Instructions

* Apply PR
* `yarn start`
* Go to `/setup/videopress`
* Follow the steps
* ✅ You should get redirected to the checkout page with a premium plan, no plan page should be displayed during the flow
